### PR TITLE
Adapt E2E test documentation

### PR DESCRIPTION
**Pull request by c0d1ngm0nk3y**
Added some pitfalls we encountered while trying to run e2e tests locally on mac.

### CODE REVIEW

**Comment by c0d1ngm0nk3y** [comment]
I signed it!

File: tests/e2e/UsingGKE.md [hunk 1]
```diff
@@ -56,12 +56,19 @@ kubectl create clusterrolebinding myname-cluster-admin-binding  --clusterrole=cl
 
 **Option 1:** Build your own images.
 
+On MacOS, you need to set the target operating system before building the images
+
+```
+export GOOS=linux
```
[the remark this answers is not in the document]
Comment (c0d1ngm0nk3y) [reply tests/e2e/UsingGKE.md]: Good point. I will change it. It worked for us, but probably because we use the correct args for the e2e tests.

**Comment by c0d1ngm0nk3y** [comment]
/assign @nmittler

Is there anything I can do on this PR?

File: tests/e2e/UsingGKE.md [hunk 2]
```diff
@@ -62,6 +62,7 @@ kubectl create clusterrolebinding myname-cluster-admin-binding  --clusterrole=cl
 cat .istiorc.mk
 HUB=costinm
 TAG=mybranch
+GS_BUCKET=mybucket
```
Comment (nmittler) [inline tests/e2e/UsingGKE.md]: why this?

Comment (c0d1ngm0nk3y) [reply tests/e2e/UsingGKE.md]: I have updated the comment to hopefully make it clearer. If not set, the GS_BUCKET *istio-artifacts* will be used (I think for the istioctl) and I do not have write permissions to that bucket.

**Comment by c0d1ngm0nk3y** [comment]
> So there's good news and bad news.
> 
> 👍 The good news is that everyone that needs to sign a CLA (the pull request submitter and all commit authors) have done so. Everything is all good there.
> 
> 😕 The bad news is that it appears that one or more commits were authored or co-authored by someone other than the pull request submitter. We need to confirm that all authors are ok with their commits being contributed to this project. Please have them confirm that here in the pull request.
> 
> _Note to project maintainer: This is a terminal state, meaning the `cla/google` commit status will not change from this state. It's up to you to confirm consent of all the commit author(s), set the `cla` label to `yes` (if enabled on your project), and then merge this pull request when appropriate._

How to "confirm that all authors are ok"? In this case there were only two authors. We have this problem with all the PRs we create. Any advise what we can/should do?

**Comment by loewenstein** [comment]
@c0d1ngm0nk3y I suppose all authors state there consent and then ping @hklai. Google cla devs hopefully work on a better solution soon.

**Comment by c0d1ngm0nk3y** [comment]
@hklai Do you know what the problem with the cla label is? Are we doing something wrong? Can we open a ticket to the bot? I think it does not scale if we always need a human overwrite (@loewenstein)

**Comment by hklai** [comment]
@c0d1ngm0nk3y it is a known problem with the CLA bot when a PR contains commits from multuple authors.
I have talked with the team responsible for it. They will hopefully prioritize a fix soon.

In the mean time, feel free to ping me directly for manual CLA override.

**Comment by c0d1ngm0nk3y** [comment]
@nmittler Could I clarify your review comment or is there still something to do? I wonder how to continue with this.

**Comment by nmittler** [comment]
@c0d1ngm0nk3y sorry this fell off my radar.

/approve
/lgtm

**Comment by c0d1ngm0nk3y** [comment]
@nmittler No problem. I was just wondering. I am still a bit unsure with the PR process within istio. What is the state now? It states that 1 approving review would be needed to automatically merge this, but isn't your 'lgtm' such a review?
I have read the process description, but maybe I missed something. Some hint would be highly appreciated :)

**Comment by nmittler** [comment]
@c0d1ngm0nk3y the master branch has been frozen while we work on the 1.1 release. I believe it should be unfrozen soon (possibly today?).  @hklai can you confirm?

**Comment by hklai** [comment]
Yup it has been unlocked.

**Pull request merged**